### PR TITLE
fix(NcAppNavigation): add border-right

### PR DESCRIPTION
**Pull request by marcoambrosini**

### CODE REVIEW

File: src/components/NcAppNavigation/NcAppNavigation.vue [hunk 1]
```diff
@@ -342,6 +342,7 @@ export default {
 	background-color: var(--color-main-background-blur, var(--color-main-background));
 	-webkit-backdrop-filter: var(--filter-background-blur, none);
 	backdrop-filter: var(--filter-background-blur, none);
+	border-right: 1px solid var(--color-border);
```
Comment (susnux) [inline src/components/NcAppNavigation/NcAppNavigation.vue:345]:
```suggestion
	border-inline-end: 1px solid var(--color-border);
```

**Review by susnux — APPROVED** [review]

**Comment by susnux** [comment]
The commit sign-off needs to be fixed, otherwise good!

**Comment by ShGKme** [comment]
Before | After
---|---
image | image
image | image
image | image
image | image

**Review by ShGKme — APPROVED** [review]
Better, but for me it makes it even less easy to find the toggle button because not it is visually out of the navigation.

**Comment by susnux** [comment]
> for me it makes it even less easy to find the toggle button because not it is visually out of the navigation.

I think as we overlay the content a shadow might be better suited:

[Bildschirmaufnahme_20240815_121458.webm](https://github.com/user-attachments/assets/27dfcaed-f7ce-40ad-87ee-98ba91f743e8)

But still this PR makes things lot better :)

**Comment by ShGKme** [comment]
Merge?

**Comment by ShGKme** [comment]
Fixed DCO and squashed

**Comment by ShGKme** [comment]
Merge?

**Pull request merged**